src/js/bootstrap-slider.js

```diff
@@ -1682,28 +1691,29 @@ const windowIsDefined = (typeof window === "object");
 
 				return false;
 			},
+			_setValues: function(index, val) {
+				var comp = index === 0 ? 0 : 100;
```

Comment:
could you use an ES6 `const` declaration instead of `var`? also, wrap the conditional in parenthesis to make it stand out more easily